gene2phenotype_project/gene2phenotype_app/serializers.py

```diff
@@ -1104,16 +1110,42 @@ def create(self, validated_data):
                     raise serializers.ValidationError({"message": f"Data already submited to G2P '{lgd_obj.stable_id.stable_id}'"})
                 else:
                     raise serializers.ValidationError({"message": f"This is an old G2P record '{lgd_obj.stable_id.stable_id}'"})
```

Comment:
`lgd_obj` is a queryset, so this should be updated to: `lgd_obj.first().stable_id.stable_id`

Reply:
Yes did not make any changes to this bit of the existing code, i think it is for publish part but will also be done differently, just added comments to it so I can remember to make changes to it in the future